ERCS/erc-7620.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 7620
```

Comment:
EIP numbers are assigned by EIP/RIP/ERC editors/associates.  7620 has already been assigned to an EIP.